.gitpod.yml

```diff
@@ -11,4 +11,4 @@ ports:
 
 tasks:
 - init: ./aero.py --no-run
-  command: ./aero.py --only-run
+  command: ./aero.py --only-run --memory 32G
```

Comment:
Will this much memory be available in GitPod for every instance?

Reply:
64 gigs of RAM is available in every Gitpod workspace. cargo uses about 25 when it's building aero